docs/transform.md

```diff
@@ -7,33 +7,40 @@ Transforms map or filter the input or output of a child process. They are define
 ```js
 import {execa} from 'execa';
 
-const transform = async function * (chunks) {
-	for await (const chunk of chunks) {
-		yield chunk.toUpperCase();
+const transform = async function * (lines) {
+	for await (const line of lines) {
+		const prefix = line.includes('error') ? 'ERROR' : 'INFO'
+		yield `${prefix}: ${line}`
 	}
 };
 
 const {stdout} = await execa('echo', ['hello'], {stdout: transform});
 console.log(stdout); // HELLO
 ```
 
+Alternatively, a `{ transform }` plain object can also be passed.
```

Comment:
I don't see the point in documenting this. It's only there for the `binary` option and that option is already documented further down.

Reply:
Yes, I was hesitating too whether to include this.
Fixed in 51881563dec1a20ad9d336d0c20c49908c887121